src/redux/slice/userSlice.ts

```diff
@@ -175,6 +168,22 @@ const authSlice = createSlice({
         };
         state.role = '';
       })
+      .addMatcher(
```

Comment:
try use for [Matchers ](https://redux-toolkit.js.org/api/matching-utilities#ispending)